whois-client/src/main/java/net/ripe/db/whois/common/apiKey/OAuthSession.java

```diff
@@ -63,4 +63,31 @@ public String toString() {
                 .add("scopes", scopes)
```

Comment:
why this class is not a record?

Reply:
Why you want to make it a record? This will be JSON parsed  once proxy layer is implemented.

Reply:
no, I don't care if it is a record or not.
Just wander why it is no a record...